R/guess_projection.R

```diff
@@ -45,8 +45,8 @@
 #' @importFrom leaflet leaflet addTiles addCircleMarkers addLayersControl
 #' addLegend setView colorFactor layersControlOptions
 #' @importFrom assertthat is.string is.flag noNA has_name
-#' @importFrom dplyr sample_n
-#' @importFrom sp CRS
+#' @importFrom dplyr slice_sample
+#' @importFrom sf st_crs
 #'
 guess_projection <- function(df, col_long, col_lat, belgium = TRUE,
                      projections = c("epsg:4326", "epsg:31370",
```

Comment:
Probably works, but should be capitalized: `EPSG:4326` etc

Reply:
done in [c5556a5](https://github.com/inbo/inborutils/pull/115/commits/c5556a5a5d1f77c5447ee6729f374946390d64a6)